apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -414,6 +417,22 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
         setIsLoading(false);
     }, [ template ]);
 
+    /**
+     * Check whether the application is an M2M Application
+     */
+    useEffect(() => {
+        if (!template?.id || !M2MApplicationTemplate?.id) {
+            setIsLoading(false);
```

Comment:
shouldn't this be `setLoading(true)` ?

Reply:
Yes. 
Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b